src/packages/external-event/external-event.ts

```diff
@@ -0,0 +1,136 @@
+import type { Express, Request, Response } from 'express';
+import type { ExternalEventTypeInsertInput } from '../../types/external-event.js';
+import Ajv from 'ajv';
+import { getEnv } from '../../env.js';
+import getLogger from '../../logger.js';
+import gql from './gql.js';
+import { HasuraError } from '../../types/hasura.js';
+
+const logger = getLogger('packages/external-event/external-event');
+const { HASURA_API_URL } = getEnv();
+const GQL_API_URL = `${HASURA_API_URL}/v1/graphql`;
+
+const ajv = new Ajv();
+
+async function uploadExternalEventType(req: Request, res: Response) {
+  logger.info(`POST /uploadExternalEventType: Entering function...`);
+  const authorizationHeader = req.get('authorization');
+  logger.info(`POST /uploadExternalEventType: ${authorizationHeader}`);
```

Comment:
Remove this, you should never be logging user credentials like this.

Reply:
Sorry - missed this from cleaning up some testing debugs, removed now!